docs/README.md

```diff
@@ -2,95 +2,103 @@
 |----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `IONOS_USERNAME`     | Specify the username used to login, to authenticate against the IONOS Cloud API                                                                                                                                                 | 
-| `IONOS_PASSWORD`     | Specify the password used to login, to authenticate against the IONOS Cloud API                                                                                                                                                 | 
-| `IONOS_TOKEN`        | Specify the token used to login, if a token is being used instead of username and password                                                                                                                                      |
-| `IONOS_API_URL`      | Specify the API URL. It will overwrite the API endpoint default value `api.ionos.com`. Note: the host URL does not contain the `/cloudapi/v6` path, so it should _ not_ be included in the `IONOS_API_URL` environment variable |
+| ``IONOS_USERNAME``     | Provide the username used to login to authenticate against the IONOS Cloud API.                                                                                                                                                | 
+| ``IONOS_PASSWORD``     | Provide the password used to login to authenticate against the IONOS Cloud API.                                                                                                                                                 | 
+| ``IONOS_TOKEN``        | Provide the token used to login if a token is being used instead of **username** and **password** .                                                                                                                                     |
+| ``IONOS_API_URL``      | Provide the API URL. It will overwrite the API endpoint default value `api.ionos.com`. |
 
-### Create Provider Secret
 
-- Using username and password:
+{% hint style="info" %}
+**Note:** The host URL does not contain the ``/cloudapi/v6`` path, so it should not be included in the ``IONOS_API_URL`` environment variable.
+{% endhint %}
 
-```bash
-export BASE64_PW=$(echo -n "${IONOS_PASSWORD}" | base64)
-kubectl create secret generic --namespace crossplane-system example-provider-secret --from-literal=credentials="{\"user\":\"${IONOS_USERNAME}\",\"password\":\"${BASE64_PW}\"}"
-```
+### Create provider secret
+
+To create the provider secret, you can use either of the following methods:
 
-- Using token:
+* [<mark style="color:blue;">Using username and password</mark>](#using-username-and-password)
+* [<mark style="color:blue;">Using token</mark>](#using-token)
```

Comment:
Maybe we can specify that we recommend using the token, given the upcoming deprecation of the basic auth

Reply:
Added a note for this :)